compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1235,6 +1235,9 @@ error.key.specifier.mismatch.with.key.constraint = \
 error.invalid.key.constraint.provided.for.access = \
   invalid key constraint provided for member access. key constraint expected with type ''{0}''
 
+error.cannot.update.table.using.member.access.lvexpr = \
+  cannot update ''{0}'' using indexing
```

Comment:
@lasinicl I don't think above error message is correct . It is not an `indexing` based access.
Shall we update it as "cannot update ''{0}'' with member access expression"?  WDYT?

Reply:
Fixed in 42ca1bb